internal/common/settings.go

```diff
@@ -27,13 +27,19 @@ type Settings struct {
 }
 
 type SourceOfTrustSettings struct {
-	File *FileTrustSourceSettings
+	File  *FileTrustSourceSettings
+	Spire *SpireTrustSourceSettings
 }
 
 type FileTrustSourceSettings struct {
 	Domains map[string][]string
 }
 
+type SpireTrustSourceSettings struct {
+	URLs         map[string]string
+	CertLocation string
```

Comment:
I'm not sure these names tell the full story of what they are.

`URLs` is a map of domain --> spire agent endpoint
and
`CertLocation` is the path to a JSON file containing a map of domain --> cert as last provided by Spire. (ie, local storage of what Spire last gave us)

Reply:
How's this?